lang/cs/Org.Apache.REEF.IMRU/Org.Apache.REEF.IMRU.csproj

```diff
@@ -57,6 +58,7 @@ under the License.
     <Compile Include="API\IMRUUpdateConfiguration.cs" />
     <Compile Include="API\IUpdateFunction.cs" />
     <Compile Include="API\IPerMapperConfigGenerator.cs" />
+    <Compile Include="API\JobCancellationDetectorAlwaysFalse.cs" />
```

Comment:
hmm, I moved it under driver namespace now, but not sure if it's correct:
 we have circular reference b/w folders now Driver depends on API to use the interface, while API depends on Driver for default implementation ....
it doesn't create problems as all this internal in the same project....but it rubs me in wrong way

---
In reply to: [86018302](https://github.com/apache/reef/pull/1161#discussion_r86018302) [](ancestors = 86018302)